dotnet/src/SemanticKernel.Core/Functions/FunctionCollection.cs

```diff
@@ -76,6 +76,11 @@ public IFunctionCollection AddFunction(ISKFunction functionInstance)
     /// <inheritdoc/>
     public ISKFunction GetFunction(string pluginName, string functionName)
     {
+        if (string.IsNullOrWhiteSpace(pluginName))
+        {
+            return this.GetFunction(GlobalFunctionsPluginName, functionName);
+        }
```

Comment:
nit: Maybe we can simplify and re-assign `pluginName` instead of calling same function again. In this case we won't increase execution stack and avoid potential infinite recursion (I know it's rare case to happen here, but still :) ).
```suggestion
        pluginName = !string.IsNullOrWhiteSpace(pluginName) ?  pluginName : GlobalFunctionsPluginName;
```